roles/mytd/templates/nginx_default_config.j2

```diff
@@ -0,0 +1,85 @@
+user www-data;
```

Comment:
* Use the `/etc/nginx/sites-available/default` for templating
* Define playbook parameter `port`
* Use the `port` parameter in the template